docs/tasks/debug-application-cluster/monitor-node-health.md

```diff
@@ -56,54 +56,16 @@ customized node problems.
 
 * **Step 1:** Create `node-problem-detector.yaml`:
```

Comment:
The pattern we try to follow is this:
Don't have the reader create the yaml file.
Show the yaml file.
In the command, give the path to the yaml file.
Here's an example from another topic:
`kubectl create -f https://k8s.io/docs/tasks/configure-pod-container/memory-request-limit.yaml`